packages/devtools/devtools/vite.config.ts

```diff
@@ -62,7 +49,14 @@ export default defineConfig({
     }
   },
   plugins: [
-    ConfigPlugin(),
+    ConfigPlugin({
+      env: [
+        'DX_ENVIRONMENT',
+        'IPDATA_API_KEY',
+        'SENTRY_DESTINATION',
+        'TELEMETRY_API_KEY'
```

Comment:
unless these are standard env keys then prefix with KUBE_ or DX_?

Reply:
yeah, I think if we're going to integrate telemetry calls into client when we do that we should update the name of the variable it's looking for to be more dxos-specific :+1: